topics/sequence-analysis/tutorials/genetic-map-rad-seq/tutorial.md

```diff
@@ -40,28 +40,28 @@ The original data is available at [STACKS website](https://creskolab.uoregon.edu
 >    > * Open the Galaxy Upload Manager
 >    > * Select **Paste/Fetch Data**
 >    > * Paste the following links into the text field
->    >     * https://cesgo.genouest.org/resources/375/download/female.fa
->    >     * https://cesgo.genouest.org/resources/376/download/male.fa
->    >     * https://cesgo.genouest.org/resources/377/download/progeny_1.fa
->    >     * https://cesgo.genouest.org/resources/378/download/progeny_2.fa
->    >     * https://cesgo.genouest.org/resources/379/download/progeny_3.fa
->    >     * https://cesgo.genouest.org/resources/380/download/progeny_4.fa
->    >     * https://cesgo.genouest.org/resources/381/download/progeny_5.fa
->    >     * https://cesgo.genouest.org/resources/382/download/progeny_6.fa
->    >     * https://cesgo.genouest.org/resources/383/download/progeny_7.fa
->    >     * https://cesgo.genouest.org/resources/384/download/progeny_8.fa
->    >     * https://cesgo.genouest.org/resources/385/download/progeny_9.fa
->    >     * https://cesgo.genouest.org/resources/386/download/progeny_10.fa
->    >     * https://cesgo.genouest.org/resources/387/download/progeny_11.fa
->    >     * https://cesgo.genouest.org/resources/388/download/progeny_12.fa
->    >     * https://cesgo.genouest.org/resources/389/download/progeny_13.fa
->    >     * https://cesgo.genouest.org/resources/390/download/progeny_14.fa
->    >     * https://cesgo.genouest.org/resources/391/download/progeny_15.fa
->    >     * https://cesgo.genouest.org/resources/392/download/progeny_16.fa
->    >     * https://cesgo.genouest.org/resources/393/download/progeny_17.fa
->    >     * https://cesgo.genouest.org/resources/394/download/progeny_18.fa
->    >     * https://cesgo.genouest.org/resources/395/download/progeny_19.fa
->    >     * https://cesgo.genouest.org/resources/417/download/progeny_20.fa
+>    >     * https://zenodo.org/record/1219888/files/female
+>    >     * https://zenodo.org/record/1219888/files/male
+>    >     * https://zenodo.org/record/1219888/files/progeny_1
+>    >     * https://zenodo.org/record/1219888/files/progeny_2
+>    >     * https://zenodo.org/record/1219888/files/progeny_3
+>    >     * https://zenodo.org/record/1219888/files/progeny_4
+>    >     * https://zenodo.org/record/1219888/files/progeny_5
+>    >     * https://zenodo.org/record/1219888/files/progeny_6
+>    >     * https://zenodo.org/record/1219888/files/progeny_7
+>    >     * https://zenodo.org/record/1219888/files/progeny_8
+>    >     * https://zenodo.org/record/1219888/files/progeny_9.fa
```

Comment:
@galaxy-e @yvanlebras The fa extension is wrong I think. You have not uploaded them with the fa extension imho.